File: C/PRAK504-2210817210005-MFithra_Siroja.c
#include <stdio.h>
int reverse(int nilai){
    int sisa; 
    int balik=0;
    while (nilai!= 0){
        sisa=nilai %10;
        balik=balik*10 + sisa;
        nilai= nilai/10;    
    }
    return balik;
}
int main() {
int A, B;
scanf("%d %d",&A,&B);
A=reverse(A);
B=reverse(B);
int C =A+B;
printf("%d" ,reverse(C));
}

File: Python/PRAK504-2210817210005-MFithra_Siroja.py
def reverse(kebalikan):
    nilai=0
    while(kebalikan !=0):
        nilai=nilai*10
        nilai=nilai+kebalikan %10
        kebalikan //=10
    return nilai
A,B= input().split()
A=reverse(int(A))
B=reverse(int(B))
C=A+B
print(reverse(C))
    